README.md

```diff
@@ -216,84 +216,24 @@ SELECT
 FROM users
 ```
 
-### Customization
```

Comment:
Instead of deleting this section should we modify it to show how one can create a custom dialect?